frontend/packages/network-attachment-definition-plugin/src/components/TechPreview/TechPreview.scss

```diff
@@ -0,0 +1,6 @@
+.TechPreviewLabel {
```

Comment:
Is there a reason why you're deviating from the standard CSS naming convention here?

Reply:
changed thanks :)